src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -362,11 +365,17 @@ private TextSpan[] Deserialize(string[] v)
             }).ToArray();
         }
 
-        public void GoToDefinition()
-            => _editorInProc.GoToDefinition();
+        public void GoToDefinition(string viewName)
+        {
+            _editorInProc.GoToDefinition();
+            _editorInProc.WaitForActiveView(viewName);
```

Comment:
LSP goto definition is async so we need to wait until the tab is opened.  this shouldn't affect local tests as that tab will be opened immediately.

Reply:
100ms doesn't seem long enough here.  i coudl easily see things failing on CI because the machine is doing just a little bit of work, and can't open the window in time.